runner.py

```diff
@@ -47,6 +49,30 @@ def load_index():
     config.rag_index
 
 
+# NOTE: Be aware this is irreversible step, as it modifies the certifi
+# store. To restore original certs, the certifi lib needs to be
+# reinstalled (or extra cert manually removed from the file).
+def add_ca_to_certifi(cert_path: str, certifi_cert_location=certifi.where()) -> None:
+    """Add a certificate to the certifi store."""
+    print(f"Certifi store location: '{certifi_cert_location}'")
+    print(f"Adding certificate '{cert_path}' to certifi store")
+    with open(cert_path, "rb") as certificate_file:
```

Comment:
are not these files regular text files?

Reply:
Yes, but the cryptography functions work with bytes.